### PR TITLE
Doc: Updating intro to avd and ansible diagrams

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

Updating diagrams to be more in the same style as our examples.

## How to test

View the generated site

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/_media/getting-started/collections.svg
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/_media/getting-started/collections.svg:1]: This is missing CVP and network nodes compared to the old drawing

Comment (carlbuchmann) [reply ansible_collections/arista/avd/docs/_media/getting-started/collections.svg:1]: image

Comment (JulioPDX) [reply ansible_collections/arista/avd/docs/_media/getting-started/collections.svg:1]: Addressed with Carl :)

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**